SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -47,6 +47,23 @@ bool ComponentFactory::FillSchemaObject(Schema_Object* ComponentObject, UObject*
 			const FRepLayoutCmd& Cmd = Changes.RepLayout.Cmds[HandleIterator.CmdIndex];
 			const FRepParentCmd& Parent = Changes.RepLayout.Parents[Cmd.ParentIndex];
 
+#if TRACE_LIB_ACTIVE
+			if (USpatialLatencyTracer* LatencyTracer = USpatialLatencyTracer::GetTracer(NetDriver))
```

Comment:
Based on the feedback from Sami and Tencho, we shouldn't use the NetDriver to get the tracer, but some world object. Perhaps just pass the `Object`.

Reply:
Also this is a bit of a tight loop, perhaps pull the tracer construction outside the while loop.